packages/cli/src/eject/eject.js

```diff
@@ -91,7 +92,8 @@ function eject() {
       path.resolve(
         'node_modules',
         'react-native',
-        'local-cli',
+        '@react-native-community',
+        'cli',
```

Comment:
looks like we don't have a correct test for this scenario

Reply:
True. I don't even know when this command is being called.